app/controllers/api/v1/admin/server_rooms_controller.rb

```diff
@@ -14,7 +14,7 @@ def index
 
           pagy, rooms = pagy(rooms)
 
-          rooms = RunningMeetingChecker.new(rooms:).call
+          RunningMeetingChecker.new(rooms:).call
```

Comment:
Double check that `get_meetings` is running a bunch of `get_meeting_info` as well so we might as well use our service in ServerRooms for consistency.